auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -102,6 +103,19 @@ public LogoutBuilder withScheme(@NonNull String scheme) {
             return this;
         }
 
+        /**
+         * Specify a custom Redirect To Url to use to invoke the app on redirection.
```

Comment:
In the param description you're using the term **URI**, but in the builder method you use **URL**. Let's use a single term.

Reply:
I've changed the "Return To Uri" in `withScheme` method to "Return To URL". The other mentions of Uri belong to the `redirect_uri` parameter that is sent on **authentication** (!= log out) so those are correct.